kroxylicious-integration-test-support/src/main/java/io/kroxylicious/test/tester/KroxyliciousConfigUtils.java

```diff
@@ -116,7 +116,9 @@ static String bootstrapServersFor(String virtualCluster, Configuration config, S
             return c.getBootstrapAddress().toString();
         }
         else if (provider.config() instanceof SniRoutingClusterNetworkAddressConfigProviderConfig c) {
-            return new HostPort(c.getBootstrapAddress().host(), c.getAdvertisedPort()).toString();
+            String bootstrapAddressPattern = c.getBootstrapAddressPattern();
```

Comment:
we could decide to do a full deprecation cycle to surface this in the public config API, maybe it's better named `bootstrapAddressPattern` there too. 

If we want to do that, I'll do it in a subsequent PR as either way I'd put the new behaviour on the old property.